cypress/integration/user.test.js

```diff
@@ -82,7 +82,7 @@ describe('User list and items test', () => {
 
     cy.get('button:last').should('have.text', 'Активувати');
 
-    cy.get('#status').should('have.text', 'Неактивний');
+    cy.get('#status').should('have.text', 'Неактивний(-a)');
```

Comment:
Please, don't use and `id` use `data-cy` attribute on components and then query it from cypress tests

Reply:
fixed